modular_skyrat/modules/fishing/code/fishing.dm

```diff
@@ -197,3 +269,27 @@ GLOBAL_LIST_INIT(fishing_weights, list(
 	reqs = list(/obj/item/stack/rods = 2,
 				/obj/item/stack/cable_coil = 2)
 	category = CAT_MISC
+
+/datum/crafting_recipe/trash_attachment
+	name = "Magnetic Fishing Attachment"
+	result = /obj/item/fishing_focus/trash
+	reqs = list(/obj/item/forging/complete/plate = 1,
+				/obj/item/forging/complete/chain = 1)
+	category = CAT_MISC
+
+/datum/crafting_recipe/fish_attachment
+	name = "Bait Fishing Attachment"
+	result = /obj/item/fishing_focus/fish
+	reqs = list(/obj/item/food/grown = 1)
+	category = CAT_MISC
+
+/datum/crafting_recipe/ore_attachment
+	name = "Pickaxe Fishing Attachment"
+	result = /obj/item/fishing_focus/ore
+	reqs = list(/obj/item/pickaxe = 1)
+	category = CAT_MISC
```

Comment:
I feel like that's very easy to make, compared to the others, since you can get easy access to those at round-start.